wagtailio/project_styleguide/templates/patterns/includes/menu_get_started.html

```diff
@@ -0,0 +1,26 @@
+<div class="modal modal--get-started micromodal-slide" id="get-started-menu" aria-hidden="true">
+    <div class="modal__overlay" tabindex="-1" data-micromodal-close>
+        <div class="modal__container" role="dialog" aria-modal="true" {% if get_started_menu_title %}aria-labelledby="get-started-menu-title"{% endif %}>
+            <header class="modal__header">
+                <button class="modal__close-button" aria-label="Close the Get Started menu" data-micromodal-close>
+                    <svg class="modal__close-icon" aria-hidden="true">
+                        <use xlink:href="#cross"></use>
+                    </svg>
+                </button>
+            </header>
+            <main class="modal__content">
+                {% if get_started_menu_title %}
+                    <p class="modal__title intro-small" id="get-started-menu-title">{{ get_started_menu_title }}</p>
+                {% endif %}
+
+                <ul class="modal__grid modal__grid--get-started">
+                    {% for menu_item in get_started_menu_items %}
```

Comment:
When the BE for this is done I imagine it'll be out via something like `{% menu_get_started %}` however I don't believe I can mock that in the pattern library without breaking the site outside of the pattern library?

Reply:
Yeah, I agree, could you leave a comment describing this? Just for clarity.

Reply:
Good shout. I'll add one.